llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -667,6 +673,9 @@ class InnerLoopVectorizer {
   /// for cleaning the checks, if vectorization turns out unprofitable.
   GeneratedRTChecks &RTChecks;
 
+  /// Mapping of induction phis to their bypass values and bypass blocks.
+  DenseMap<PHINode *, std::pair<BasicBlock *, Value *>> InductionBypassValues;
+
```

Comment:
This is needed only for "additional" bypasses. Is there a way to avoid storing this mapping?